# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/ppc-unaligned.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some user-space apps (e.g. reiserfsprogs) do need the unaligned macros, ...

  - Rene Rebe <rene@exactcode.de>

--- ./include/asm-ppc/unaligned.h.orig	2004-10-25 12:12:02.331864008 +0200
+++ ./include/asm-ppc/unaligned.h	2004-10-25 12:12:09.347797424 +0200
@@ -1,4 +1,4 @@
-#ifdef __KERNEL__
+
 #ifndef __PPC_UNALIGNED_H
 #define __PPC_UNALIGNED_H
 
@@ -15,4 +15,4 @@
 #define put_unaligned(val, ptr) ((void)( *(ptr) = (val) ))
 
 #endif
-#endif /* __KERNEL__ */
+
